backend/tests/v2/factories/uniqueness_quotients.py

```diff
@@ -0,0 +1,56 @@
+from decimal import Decimal
+from typing import Optional
+
+from async_factory_boy.factory.sqlalchemy import AsyncSQLAlchemyFactory
+from factory import LazyAttribute, SubFactory
+from sqlalchemy import select
+
+from app.infrastructure.database.models import UniquenessQuotient, User
+from tests.v2.factories.base import FactorySetBase
+from tests.v2.factories.users import UserFactory
+
+
+class UniquenessQuotientFactory(AsyncSQLAlchemyFactory):
+    class Meta:
+        model = UniquenessQuotient
+        sqlalchemy_session_persistence = "commit"
+
+    user = SubFactory(UserFactory)
+    epoch = LazyAttribute(lambda _: 1)  # Default epoch
+    score = LazyAttribute(lambda _: str(Decimal("1.0")))  # Default score
```

Comment:
You don't have to use LazyAttribute here, I guess. You can only hardcode the default value.